### PR TITLE
August MFIS Update

### PR DESCRIPTION
**Pull request by npstrayer**

### CODE REVIEW

**Comment by npstrayer** [comment]
@NeighborhoodInfoDC/review

**Comment by ptatian** [comment]
@npstrayer I will review.

File: Prog/MFIS/MFIS_2017_08.sas [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/**************************************************************************
+ Program:  MFIS_yyyy_mm.sas
+ Library:  HUD
+ Project:  NeighborhoodInfo DC
+ Author:
```
Comment (ptatian) [inline Prog/MFIS/MFIS_2017_08.sas]: You should fill in the header information for Author: and Created: (date).

File: Prog/MFIS/MFIS_2017_08.sas [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/**************************************************************************
+ Program:  MFIS_yyyy_mm.sas
```
Comment (ptatian) [inline Prog/MFIS/MFIS_2017_08.sas]: You should update the header with the actual program name (MFIS_2017_08.sas).

**Comment by ptatian** [comment]
@npstrayer Actually, sorry. Hold up for a moment please.

**Comment by ptatian** [comment]
@npstrayer OK, I sorted it out. As I mentioned, you had also committed the MFIS program under the other branch (update-#117), which would have created a potential conflict. I solved it by just deleting tha program on the other branch. 

You can now go ahead and proceed with the changes I requested and finalize this pull request. **One more thing, however.** The program filename isn't quite right. If the file extract date is 7/31/17, then the program name should be **MFIS_2017_07.sas**. So, please rename the program file. You will also have to commit that change before merging and finalizing. 

Please let me know if you have any questions. I'm continuing my review of the other pull requests.

**Pull request merged**